pyquil/api/_qac.py

```diff
@@ -22,13 +22,13 @@
 
 
 class AbstractCompiler(ABC):
-    @abstractmethod
     def get_version_info(self) -> dict:
         """
         Return version information for this compiler and its dependencies.
 
         :return: Dictionary of version information.
         """
+        raise NotImplementedError
```

Comment:
raise NotImplementedError()

Reply:
I'll take that back. Seems that one can skip the object creation parentheses when raising an exception. https://docs.python.org/3/tutorial/errors.html section 8.4

Reply:
TIL